tests/migration_test.go

```diff
@@ -2330,12 +2320,14 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 			It("[test_id:1862][posneg:negative]should reject migrations for a non-migratable vmi", func() {
 				// Start the VirtualMachineInstance with the PVC attached
 
-				vmi, _ := tests.NewRandomVirtualMachineInstanceWithBlockDisk(cd.DataVolumeImportUrlForContainerDisk(cd.ContainerDiskAlpine), util.NamespaceTestDefault, k8sv1.ReadWriteOnce)
+				vmi, _ := tests.NewRandomVirtualMachineInstanceWithBlockDisk(cd.DataVolumeImportUrlForContainerDisk(cd.ContainerDiskAlpineExtended), util.NamespaceTestDefault, k8sv1.ReadWriteOnce)
 				vmi.Spec.Hostname = string(cd.ContainerDiskAlpine)
 				vmi = runVMIAndExpectLaunch(vmi, 180)
 
 				By("Checking that the VirtualMachineInstance console has expected output")
-				Expect(libnet.WithAlpineConfig(console.LoginToAlpine)(vmi)).To(Succeed())
+				networkData, _ := libnet.CreateDefaultCloudInitNetworkData()
+				tests.AddCloudInitNoCloudData(vmi, "disk1", "", networkData, false)
```

Comment:
Isn't it too late to do this here? It needs to be done before the `runVMIAndExpectLaunch`, no?

Reply:
It is a wrong copy paste, those two lines have to be removed.
Thanks for noticing it!